File: src/main/java/com/marshall/guy/MealPlan.java
package com.marshall.guy;

public class MealPlan {
    public DayPlan monday;
    public DayPlan tuesday;
    public DayPlan wednesday;
    public DayPlan thursday;
    public DayPlan friday;
    public DayPlan saturday;
    public DayPlan sunday;

    public MealPlan(DayPlan monday, DayPlan tuesday, DayPlan wednesday, DayPlan thursday, DayPlan friday, DayPlan saturday, DayPlan sunday) {
        this.monday = monday;
        this.tuesday = tuesday;
        this.wednesday = wednesday;
        this.thursday = thursday;
        this.friday = friday;
        this.saturday = saturday;
        this.sunday = sunday;
    }
}

File: src/meal_plan.rs
use crate::day_plan::DayPlan;

#[derive(Debug)]
pub(crate) struct MealPlan<'a> {
    pub(crate) monday: DayPlan<'a>,
    pub(crate) tuesday: DayPlan<'a>,
    pub(crate) wednesday: DayPlan<'a>,
    pub(crate) thursday: DayPlan<'a>,
    pub(crate) friday: DayPlan<'a>,
    pub(crate) saturday: DayPlan<'a>,
    pub(crate) sunday: DayPlan<'a>,
}
